webapp/src/components/EditEstatePage/EstateSelect/EstateSelect.js

```diff
@@ -201,52 +212,42 @@ export default class EstateSelect extends React.PureComponent {
                       max: MAX_PARCELS_PER_TX
                     })}
                   />
-                  <p className="warning-parcels-limit" />
                 </Grid.Column>
               </Grid.Row>
             ) : null}
 
             <Grid.Row>
-              <Grid.Column width={isCreation ? 16 : 8}>
+              <Grid.Column width={16}>
                 <Header size="large">
-                  <p>
-                    {isCreation
-                      ? t('estate_select.new_selection')
-                      : t('estate_select.edit_selection')}
-                  </p>
+                  {isCreation
+                    ? t('estate_select.new_selection')
+                    : t('estate_select.edit_selection')}
                 </Header>
-              </Grid.Column>
-              {!isCreation &&
-                isOwner(wallet, estate.id) && (
-                  <Grid.Column width={8} className="parcels-included-headline">
+                {!isCreation &&
+                  isOwner(wallet, estate.id) && (
                     <Button
                       size="tiny"
-                      className="link"
+                      className="link dissolve-button"
                       onClick={onDeleteEstate}
                     >
                       <Icon name="trash" />
-                      {t('estate_detail.dissolve')}{' '}
+                      {t('estate_detail.dissolve').toUpperCase()}
                     </Button>
-                  </Grid.Column>
-                )}
-              <Grid.Column width={16} className="parcels-included">
+                  )}
                 <p className="parcels-included-description">
                   {t('estate_select.description')}
                 </p>
-                {allParcels &&
-                  parcels.map(({ x, y }) => {
-                    const parcel = allParcels[buildCoordinate(x, y)]
-                    return parcel ? (
-                      <ParcelCoord key={parcel.id} parcel={parcel} />
-                    ) : null
-                  })}
               </Grid.Column>
+              {allParcels && (
+                <Grid.Column width={16}>
+                  <ParcelCoords
+                    parcels={this.getEstateParcels()}
+                    isCollapsable={false}
+                  />
+                </Grid.Column>
+              )}
               {canEdit && (
-                <Grid.Column
-                  className="parcel-actions-container"
-                  computer={8}
-                  tablet={16}
-                >
+                <Grid.Column width={14}>
```

Comment:
Should it be `16`?

16

<img width="847" alt="screen shot 2018-11-27 at 15 40 03" src="https://user-images.githubusercontent.com/7549152/49103670-c2901100-f25a-11e8-9919-4ea18bf42dce.png">

14

<img width="845" alt="screen shot 2018-11-27 at 15 40 08" src="https://user-images.githubusercontent.com/7549152/49103681-c91e8880-f25a-11e8-86c5-f93cec04f84f.png">